src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="*"
+      method="post"
+      data-qa="big"
+      class="bigForm"
+    >
+      <label for="search" class="labelBig">
+        <input
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="input"
+          data-qa="keypress"
+        >
+      </label>
+      <img
+        src="images/Search.svg"
+        alt="big search"
+        class="images"
+      >
+    </form>
+    <form
+      action="*"
+      method="post"
+      data-qa="small"
+      class="smallForm"
```

Comment:
You don't need to have two separated classes as `bigForm` and `smallForm`
You can set one class `form` for both elements (collect all common rules for these elements there) and then add extra class `form--small` for the form that has differences
Repeat the same all over the document
```suggestion
      class="form form--small"
```